src/mswPact.ts

```diff
@@ -51,8 +51,10 @@ export const setupMswPact = ({
   };
 
   logGroup(`Adapter enabled${options.debug ? ' on debug mode' : ''}`);
-  console.log('options:', options);
-  console.groupEnd();
```

Comment:
Missing to close the console's group when not in debug mode

Reply:
Ahh yes, shall I else off the if options.debug to close the log group if not debug?

Reply:
updated now 👍🏾 thanks man